### ISSUE TITLE
Changes to prepare menu in frm main

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#6810]
This issue brings together the aspects of issues #6807, #6797 and #6791 that give changes to the Prepare menu.  Those issues remain, because this task is just part of them.  Before @mmulahvictor makes the changes I would like @shadrackkibet confirmation that they are sensible.
1. Move Prepare > Column: Calculate > Duplicate Column dialogue moved to the Prepare > Data Frame menu.  It should go just under the Rename Column dialogue, which is then its same position in the right-click menu.  
2. Change the Prepare > Column: Calculate name in the menu to Prepare > Column: Numeric.  (The operations there are essentially all mainly on numeric variables, and this is neat in explaining that these 3 menus refer in turn to Numeric, Factor and Character-type data!
3. Change the name of the Rank dialogue in the Prepare > Column: Numeric > Rank menu to Transform.  This is discussed in #6791.  (It will include the rank dialogue, so just the change in name is needed for now.
4. In the Prepare > Data Object menu make the first item (Data Frame Metadata) and the last 3 items - from Rename Metadata downwards all invisible.  This is discussed in #6797 
5. Finally (as I think it affects frmMain) make a blank dialogue to start the work on Reorder Data Frames.  This is for the work identified in #6797

**Comment by shadrackkibet** [comment]
@rdstern I agree with the changes. Just one addition, we have some pattern, Column:..., the calculator looks different, I suggest we add a separator below it. How about "Column: Calculator" then we don't add a separator? What do you think?
![image](https://user-images.githubusercontent.com/26301973/135428750-26c5e668-cfef-4540-8426-b0cb48d3cc2d.png)


@mmulahvictor once you are done with the following changes please post the screenshots showing the previous and the current look in the PR.

**Comment by shadrackkibet** [comment]
In addition, hide the following two menu items. Haven't seen any issue with an attempt to suggest enabling?
![image](https://user-images.githubusercontent.com/26301973/135430024-128a7b24-cfdb-4cc2-9d0a-13e5818873cb.png)

**Comment by rdstern** [comment]
I like the idea of Column: Calculator as the menu topic.

**Comment by rdstern** [comment]
While we have that whole long list we have the Prepare > Check Data > Visualise dialogue.  It is also appropriate at the top of the Describe One Variable menu.- so could this be undertaken.  See #6794   Please also add the help context number in the code if possible.  Then #6794 can be closed.
